source/content/guides/wordpress-composer/04-bedrock-composer.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Using Bedrock to Create Your WordPress Pantheon Site
+subtitle: Create a Composer-managed WordPress Site with Bedrock
+description: Learn more about Bedrock for Composer-managed WordPress sites.
+categories: [develop]
+tags: [wordpress]
+contributors: [whitneymeredith]
+layout: guide
+showtoc: true
+permalink: docs/guides/wordpress-composer/bedrock-composer
+anchorid: bedrock-composer
+---
+
+[Composer](https://getcomposer.org/) does not fully support WordPress, however, Bedrock is a central feature of Composer. [Bedrock](https://roots.io/bedrock/) is a custom WordPress project structure that helps simplify working on large projects with multiple plugins and custom code. 
+
+## Create a New Pantheon WordPress Site with Bedrock
```

Comment:
It feels like a step is missing here. How do users get the site to start out? When using `composer create-project` it is expected that this is the first step in setting up a site in an empty repository. This isn't possible if we've created a site first as described in the "Create a Basic Composer-managed WordPress Site" doc. Even if we deleted everything from that site after creating and cloning, we'd still have the `.git` directory left (or we'd break the ability to commit and push).

There is an "Empty WordPress" upstream that was created for this purpose, is the intention that they start out with that? (I suspect that would still run into issues if cloned because the directory would still not be empty when running `composer create-project`.)